package com.github.soup.follow.exception

class NotFoundFollowAuthorityException : RuntimeException(
    "해당 팔로우에 대한 권한이 없습니다."
)